### ISSUE TITLE
Fix Hydra lock-on issue

### ISSUE THREAD

**Issue by ElPendejoN** [multitheftauto/mtasa-blue#3081]
### Describe the bug

There is a bug that happens in every server with no script handling it, its about lock on about the hydra, basically you lock-on your fellow players but your missiles don't track them, you just shoot normal missiles, prob related to ping or another factors idk, but can it be fixed by replacing the gta sa script about it? maybe make it for online, since in gta sa it obviously works cuz its not online.

### Steps to reproduce

1. get friends
2. spawn hydras
3. try to lock-on

### Security Policy

- [X] I have read and understood the Security Policy and this issue is not security related.

**Comment by patrikjuvonen** [comment]
Sounds like duplicate of #2795

**Comment by ElPendejoN** [comment]
Are you sure? it says invincibility bug not lock-on bug

**Comment by patrikjuvonen** [comment]
> Are you sure? it says invincibility bug not lock-on bug

The invincibility in the title is a little misleading, the rockets do not get fired.

I said _sounds like,_ I am not sure if this is a duplicate, but my interpretation of these issues sounds like the same (underlying) problem. A video could also help.

**Comment by ElPendejoN** [comment]
Yeah, it's most likely a duplicate then, but ugh it's from 2022, do you think it'll be paid attention some time?

**Comment by ElPendejoN** [comment]
The videos he shows are also a slightly different from what happens in a default MTA server because it's scripted and the script says he can't shoot because he's not on lock on when he apparently is, in a default MTA server you will try to lock on but all you will shoot is normal missiles because that's the bug, it's a popular bug.
